### PR TITLE
fedora/SPECS/pam_usb.spec: add BuildRequires

### PR DESCRIPTION
**Pull request by hustliyilin**
Signed-off-by: YiLin.Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

File: fedora/SPECS/pam_usb.spec [hunk 1]
```diff
@@ -14,6 +14,7 @@ Version:   %{version}
 Release:   %{release}
 Prefix:    /usr
 Group:     System Environment/Base
+BuildRequires: rsync libudisks2-devel libxml2-devel
```
Comment (mcdope) [inline fedora/SPECS/pam_usb.spec:17]: Why rsync?

Comment (hustliyilin) [reply fedora/SPECS/pam_usb.spec:17]: Because the rsync command is called in the fedora spec @mcdope .

Comment (mcdope) [reply fedora/SPECS/pam_usb.spec:17]: Where? Not as far as I know.

Comment (mcdope) [reply fedora/SPECS/pam_usb.spec:17]: nvm, forgot about that - sorry :D 

But actually, we should change that to use regular cp instead. I don't see the need to use rsync there.

Comment (mcdope) [reply fedora/SPECS/pam_usb.spec:17]: Gonna accept the change and will create a ticket to get rid of rsync for that.

**Pull request merged**